ray-on-gke/tpu/kuberay-tpu-webhook/deployments/deployment.yaml

```diff
@@ -21,11 +43,11 @@ spec:
         app: kuberay-tpu-webhook
     spec:
       containers:
-        - image: us-docker.pkg.dev/ai-on-gke/kuberay-tpu-webhook/kuberay-tpu-webhook:v1.1
+        - image: us-docker.pkg.dev/ai-on-gke/kuberay-tpu-webhook/kuberay-tpu-webhook:v1.2
```

Comment:
I think we bump the version in a separate PR after the tag is published. This image won't be available right away right?

Reply:
There are actually two repositories in https://pantheon.corp.google.com/artifacts/docker/ai-on-gke/us/kuberay-tpu-webhook, `kuberay-tpu-webhook` contains manually pushed images (so it would have the image) while `tpu-webhook` is built from the pipeline. I agree though that it'd be better to bump it after the PR is merged, I'll take this change out of the PR.

Reply:
I that that image should still be based on release tags in this repo, what do you think?

Reply:
Sure that sounds good to me, each new image that we push can be based on a tagged commit